src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/mediation/MediationFailedCallbackHandlerTest.java

```diff
@@ -294,6 +310,29 @@ public void shouldTriggerDQDeadlineEventIfOfflineDQ() {
                 .saveCaseEvent(AUTHORISATION, claim.getCcdCaseId(), CaseEvent.DIRECTIONS_QUESTIONNAIRE_DEADLINE);
     }
 
+    @Test
+    public void testReadyForDirectionsDateForMediationUnsuccessful() {
+        clock = Clock.fixed(Instant.parse(INSTANT_EXPECTED), ZoneId.of("UTC"));
+        LocalDateTime dateTime = LocalDateTime.now(clock);
```

Comment:
No point making a `@Mock` clock and then overwriting it before ever using it. Also as this is just a reference instant you don't need a clock.
```suggestion
        var dateTime = LocalDateTime.parse(INSTANT_EXPECTED);
```

Reply:
Suggestion taken.

Reply:
Thanks! Also watch out for unused imports

Reply:
Checked. Thanks.